apps/webClient/src/views/Home/index.tsx

```diff
@@ -1,61 +1,139 @@
 import React from "react";
 import { makeStyles } from "@material-ui/styles";
-import Box from "@material-ui/core/Box";
 import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
 import Typography from "@material-ui/core/Typography";
 import { Theme } from "@material-ui/core/styles/createMuiTheme";
-import Logo from "../../components/Logo";
```

Comment:
Eventually we will use an svg component logo, we might as well keep this component, and have it use the `.png` until we get an `.svg`